packages/api/lambdas/db-indexer.js

```diff
@@ -187,29 +180,21 @@ async function indexRecord(esClient, record) {
  * @returns {Promise<Array>} array of records indexed
  */
 async function indexRecords(records) {
-  const concurrencyLimit = process.env.CONCURRENCY || 3;
-  const limit = pLimit(concurrencyLimit);
   const esClient = await Search.es();
 
-  const promises = records.map((record) => limit(() => indexRecord(esClient, record)));
-  return Promise.all(promises);
+  return pMap(
+    records,
+    (record) => indexRecord(esClient, record).catch(log.error),
```

Comment:
Some errors could be retriable?  Could we retry the actions before quit?

Reply:
There may be errors that we'd want to retry, but I think that we would want to retry them where the error occurred.  (I don't want to have a "retry everything" statement here, because there could definitely be cases where we _don't_ want to retry.

I think that, if we see specific errors that we would want to retry, we should write tickets to address that.

Reply:
Ok agreed